### PR TITLE
feat: support configuring local walrus env

### PR DESCRIPTION
**Pull request by gitlawr**
Refer to https://github.com/seal-io/docs/pull/191/files for the UX.

**Related Issue:**
https://github.com/seal-io/walrus/issues/2062

### CODE REVIEW

**Comment by thxCode** [comment]
the `--env-file` is syntactic sugar that loads the env vars from a file without specifying `--env` one by one, which means modifying `local.env` also cannot update the existing container's running options after `docker restart`. we can verify this by `docker inspect`.

if so, I suggest supporting something like this: `SERVER_foo walrus local install/reinstall`, and also give a fixed docker network to avoid the IP change after reinstall. See https://github.com/seal-io/walrus/pull/1999.

to implement this, we can filter out the env vars from `os.Environ` with `SERVER_` prefix, and lock the special env vars we need, like `SERVER_ENABLE_AUTHN` and `SERVER_SETTING_LOCAL_ENVIRONMENT_MODE`, finally, inject the filter list into `cmd.Env`.

File: pkg/cli/local/install.go [hunk 1]
```diff
@@ -77,12 +82,20 @@ func InstallLocalWalrusDockerContainer() error {
...
 		"/var/run/docker.sock:/var/run/docker.sock",
-		fmt.Sprintf("sealio/walrus:%s", getLocalWalrusTag()),
-		"walrus",
-		"--enable-authn=false",
-	)
+	}
+
+	for _, env := range opts.Env {
+		runArgs = append(runArgs, "-e", env)
+	}
```
Comment (thxCode) [inline pkg/cli/local/install.go:93]: image

should we allow a user to overwrite `SERVER_SETTING_LOCAL_ENVIRONMENT_MODE` and `SERVER_ENABLE_AUTHN` ?

Comment (gitlawr) [reply pkg/cli/local/install.go:89]: Updated.

**Review by thxCode — APPROVED** [review]
LGTM

**Review by aiwantaozi — APPROVED** [review]
LGTM

**Pull request merged**